lib/string.c

```diff
@@ -125,4 +146,4 @@ char *itoa(int num, char *str)
     reverse(str);
 
     return str;
-}
+}
```

Comment:
Newline at EOF, please, though newer GCC versions do no longer warn about the lack thereof :)